README.md

```diff
@@ -134,6 +134,27 @@ To open the prometheus web UI, once the container is up, run the below:
 $ make prometheus-ui
 ```
 
+### Start / Stop mock for rbac
+
+- To use this you need to enable RBAC into `config/configs.yaml` file:
+
+  ```yaml
+  clients:
+    rbac_enabled: True
+    rbac_base_url: http://localhost:8800/api/rbac/v1
+    rbac_timeout: 30
+  mocks:
+    my_org_id: "12345"
+    rbac:
+      account_admin: "12345"
+      account_viewer: "123456"
+  ```
+
+- Now run: `make mock-start`
+- Run the application by: `make run`
+- Make some request using: `./scripts/header.sh 12345 12345` for admin or `./scripts/header.sh 12345 123456` for viewer.
```

Comment:
I think we want to update header.sh to support usernames, so you can run:  './scripts/header.sh 12345 admin'

Reply:
meaning we'd get rid of the account id completely